packages/components/src/local/admin-message/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import moment from 'moment'
+import classNames from 'classnames'
+import Chip from '@material-ui/core/Chip'
+
+/* Import Types */
+import PropTypes from './types/props'
+
+/* Import Stylesheet */
+import styles from './styles.module.scss'
+
+/* Render component */
+export const AdminMessage: React.FC<PropTypes> = ({ message }: PropTypes) => {
+  const { details } = message
+  const { force, role, name } = details.from
+  const isOwnMessage = name === force
```

Comment:
@zukilover - I _think_ the above logic may be wrong.  I _think_ we need to compare the force that sent it to the force of the logged in player.   This could be passed into the component in props, since parent components will know it.  

So, I guess you could pass "Blue" into the Story definition.